src/main/java/com/programmers/springbootjpa/service/CustomerService.java

```diff
@@ -0,0 +1,49 @@
+package com.programmers.springbootjpa.service;
+
+import com.programmers.springbootjpa.domain.Customer;
+import com.programmers.springbootjpa.dto.CustomerCreateRequest;
+import com.programmers.springbootjpa.dto.CustomerResponse;
+import com.programmers.springbootjpa.dto.CustomerUpdateRequest;
+import com.programmers.springbootjpa.repository.CustomerRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class CustomerService {
+
+    private final CustomerRepository customerRepository;
+
+    @Transactional
+    public CustomerResponse createCustomer(CustomerCreateRequest request) {
+        Customer savedCustomer = customerRepository.save(Customer.of(request));
+
+        return CustomerResponse.of(savedCustomer);
+    }
+
+    public List<CustomerResponse> findAllCustomers() {
+        return customerRepository.findAll().stream().map(CustomerResponse::of).toList();
```

Comment:
저는 한줄에 점을 하나 찍어서 다음줄로 넘기면 가독성이 더 개선된다고 생각하여 길어질 경우 그렇게 적용하고 있는데 이 부분에 대해서는 어떻게 생각하시나요?!

```suggestion
        return customerRepository.findAll()
                                  .stream()
                                  .map(CustomerResponse::of)
                                  .toList();
```

약간 이런식으로여!

Reply:
반영 : [0b68d4b](https://github.com/prgrms-be-devcourse/springboot-jpa/pull/267/commits/0b68d4bf42fb51765e8f40b0390e3b9c0c5b919d)
- 해당 부분은 프로젝트 패키지 전체를 대상으로 Reformat Code를 진행했는데, 자동으로 바뀐 부분이더라구요 ㅠㅠ
- 다시 원래대로 바꿔놨습니다!